assets/src/web-stories-block/css/views/list.css

```diff
@@ -10,6 +10,50 @@
   z-index: 1;
 }
 
+.iframe-embed .web-stories-list.is-view-type-list .web-stories-list__story {
```

Comment:
Added so the widget preview looks reasonable.

Reply:
That's a lot of CSS to make that preview look reasonable. Is the iframe preview so bad?

Also, bit confused whether this is for the legacy widget block, or our regular block in the new block-based widgets editor.

Reply:
Without this CSS, the web stories legacy widget block looks like this. A complete mess. 

<img width="1884" alt="Screenshot 2021-07-05 at 11 12 48" src="https://user-images.githubusercontent.com/237508/124456119-40f1af80-dd82-11eb-9c41-40b85aefb6d4.png">

This is the case, where you create a widget in WP 5.7 and upgrade. The legacy widget needs styling. 

For the record, I don't like copying and pasting here, which is all this is, but I can't see a way around it.